src/azure-cli-core/azure/cli/core/auth/landing_pages/success.html

```diff
@@ -2,11 +2,29 @@
 <html>
 <head>
     <meta charset="utf-8" />
-    <meta http-equiv="refresh" content="10;url=https://docs.microsoft.com/cli/azure/">
+    <meta http-equiv="refresh" content="120;url=https://docs.microsoft.com/cli/azure/">
     <title>Login successfully</title>
+    <style>
+        body {
+            font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif; 
+        }
+
+        code {
+            font-family: 'Courier New', Courier, monospace;
+            display: inline-block;
+            font-weight: bold;
+            background-color: rgb(229, 229, 229);
+            padding: 12px 16px;
+            margin: 8px 0px;
+        }
+    </style>
 </head>
 <body>
-    <h4>You have logged into Microsoft Azure!</h4>
-    <p>You can close this window, or we will redirect you to the <a href="https://docs.microsoft.com/cli/azure/">Azure CLI documentation</a> in 10 seconds.</p>
+    <h3>You have logged into Microsoft Azure!</h3>
+    <p>You can close this window, or we will redirect you to the <a href="https://docs.microsoft.com/cli/azure/">Azure CLI documentation</a> in 2 minutes.</p>
+    <h3>Announcements</h3>
```

Comment:
This is a clever usage of MSAL Python's `success_template` feature. Good job. I guess MSAL shall better officially document this parameter/behavior, then. :-)

Reply:
One more thing. Currently, the broker is only available on Windows. Shall we keep using the old success template when Azure CLI is running on non-Windows platform, and only use the new one on Windows?

The conditional, per-platform guidance is inevitable in the long run, because we will probably need that pattern in the near future when the broker on other platforms becomes available.